### ISSUE TITLE
Implement saveTotalModel using the new frontend

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#5248]
The saveTotalModel API currently uses a precursor of the new frontend, which doesn't work correctly in many cases. It should be replaced with a new implementation based on the new frontend instead.

One idea is to instantiate a model using only the first phase of the instantiation, and then discard any model that hasn't been expanded.

**Comment by adrpo** [comment]
https://trac.openmodelica.org/OpenModelica/ticket/5248

**Comment by casella** [comment]
@adrpo, isn't Save Total using the new frontend now?

**Comment by adrpo** [comment]
I don't think so. Still using the first (or second) incarnation of the new front-end and we are at the third one :)

**Comment by casella** [comment]
I remember when the current old backend was the new backend 😄
